mods/cnc/maps/nod06a/rules.yaml

```diff
@@ -9,6 +9,13 @@ World:
 		BriefingVideo: nod6.vqa
 		StartVideo: sundial.vqa
 		LossVideo: banner.vqa
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			normal: Normal
+			tough: Real tough guy
```

Comment:
Sorry, I didn't realize that "Real touch guy" was only a thing in RA (which is silly, because that was a Commando quote). Can you please change this to `hard: Hard` and the lua check to match, for consistency with the other TD missions?

Reply:
Fixed